docs/custom-risk-types.md

```diff
@@ -0,0 +1,55 @@
+# Custom Risk Types
+
+It is possible to define a custom risk type that CLI will recognize.
+It can be a secret (e.g. an API token), a PII (Personal Identifying Information), or NIL (Non-Inclusive Language)
+
+## File Format
+
+Custom risk type is defined in an YAML file. 
+
+Here is an example file to detect GitLab PAT token:
+
+```yaml
+regex:
+  value: glpat-[a-zA-Z0-9\-_]{20}
+type: gitlab_personal_access_token
+category: secret
+description: GitLab personal access token
+precedence: strong_pattern
+```
+
+### Field Descriptions
+
+- **value**: specifies a regular expression to match the risk. Vault Radar supports golang stype regular expressions as well as PCRE
+- **type**: unique identifier for the risk type
```

Comment:
Should we explicitly mention that this should be in snake case without any spaces for the sake of consistency?